.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: If something seems to be broken then tell us here!
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Describe the bug
+A clear and concise description of what the bug is.
+
+# Tell us how to reproduce the bug
+
+Steps to reproduce:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+Minimal reproducible code example:
+
+```
+// code here
+```
+
+## Configuration
+List the version for each point below and append additional versions specific to your project
+ - Rust Version: 
+ - Forc Version: 
```

Comment:
1. Rather than embedded in the text where they have to be deleted, these should be comments.
2. Instead of being comments, they should be instructions in a form: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

These specific instructions should be accompanied by instructions on how to get them.